src/services/playlist.go

```diff
@@ -0,0 +1,41 @@
+package services
+
+import (
+	"foodloop/src/models"
+	"net/http"
+	"strconv"
+
+	"github.com/go-chi/render"
+)
+
+type FoodlistService struct{}
+
+var Foodlist FoodlistService
+
+func (*FoodlistService) GetAll(w http.ResponseWriter, r *http.Request) {
+	FoodlistServices := []models.Food{
+		{
+			FoodID:   1,
+			FoodName: "hehe",
+		},
+	}
+	render.JSON(w, r, FoodlistServices)
+}
+
+func (*FoodlistService) GetFoodlist(w http.ResponseWriter, r *http.Request) {
+	FoodlistServices := []models.Foodlist{}
+
+	id := r.Context().Value("FoodlistServiceID").(string)
```

Comment:
i think the context key used was `"foodlist"`?  Anyway, we dont have to use ctx if you feel its unnecessary. The one in user.go is just an example. Our project probably has no usecase for ctx its too small, but up to you! im fine with this